docs/nidigital/class.rst

```diff
@@ -1243,60 +1243,51 @@ frequency_counter_measure_frequency
 
 
 
-get_channel_name
-----------------
+get_channel_names
+-----------------
 
     .. py:currentmodule:: nidigital.Session
 
-    .. py:method:: get_channel_name(index)
+    .. py:method:: get_channel_names(indices)
 
-            TBD
+            Returns a list of channel names for given channel indices.
 
             
 
 
 
-            :param index:
-
-
-                
+            :param indices:
 
 
-            :type index: int
-
-            :rtype: str
-            :return:
-
-
-                    
-
-
-
-get_channel_name_from_string
-----------------------------
-
-    .. py:currentmodule:: nidigital.Session
-
-    .. py:method:: get_channel_name_from_string(index)
-
-            TBD
-
-            
-
-
-
-            :param index:
+                Specifies indices for the channels in the session.
+                Valid values are from zero to the total number of channels in the session minus one.
+                The following types and formats are supported:
+                  - Basic Sequence types
+                    - list - for example, [0, 2, 3, 1]
+                    - tuple - for example, (0, 2, 3, 1)
+                    - range - for example, range(0, 15, 2)
+                  - slice - for example, slice(0, 15, 2)
+                  - str
+                    - A comma-separated list - for example, "0,2,3,1"
+                    - A range using a hyphen - for example, "0-3"
+                    - A range using a colon - for example, "0:3"
```

Comment:
These are not very Pythonic. I'd either remove from documentation or condense into a single line to de-emphasize importance.

```
- str - for example "0", "0,2,3", "0-3"...

Reply:
> Maybe we can qualify it along the lines of "use this as a last resort"?

I wouldn't use that wording :)
If you want to keep it, we'll just make it the last option.

Reply:
I'll keep it. It's already the last option.